content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,113 @@
+
+VM support will continue to be a high priority for us in 2021, and you can expect us to see more feature
+graduation in upcoming releases.
+
+### External Istiod promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
+to include remote clusters for many applications (e.g. global distribution, high availability). By promoting this
+feature to Beta we recognize this as a must for adoption and to promote widespread-usage.
+
+Similar to changes made for Virtual Machines, the Istio community has revamped and improved our
+[multicluster](/docs/setup/install/multicluster/) installation guides:
+
+* [Before you begin](/docs/setup/install/multicluster/before-you-begin/)
+* [Install Primary-Remote](/docs/setup/install/multicluster/primary-remote/)
+* [Install Primary-Remote on different networks](/docs/setup/install/multicluster/primary-remote_multi-network/)
+* [Verify installation](/docs/setup/install/multicluster/verify/)
+
+## `service-apis` promoted to Alpha
+
+Support for [Kubernetes `service-apis`](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.8. These are marketed as the next generation of the Ingress API, and draw inspiration from
+Istio's `Gateway` and `VirtualService`. The Istio community is eager to evolve with Kubernetes and these CRDs in upcoming
+releases. To try them out, check out the [getting started documentation](/docs/tasks/traffic-management/ingress/service-apis/).
+These APIs are still evolving, so feedback is welcome!
+
+## Third-party providers in Authorization Policy
+
+Authorization Policy has been changed to allow
+[third-party providers](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-ExtensionProvider)
+using [CUSTOM](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action) actions. This allows
+Mesh Admins to use external providers to check if an incoming HTTP request is authorized. The provider must
+implement the [Envoy `ext_authz` filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter)
+authorization check either using HTTP or gRPC.
+
+### Request Classification promoted to Beta
+
+Istio continues to make mesh telemetry collection more configurable. In this release,
+[Request Classification](/docs/tasks/observability/metrics/classify-metrics/) has been promoted to Beta. This feature
+enables users to more precisely understand and monitor the traffic in their service mesh.
+
+## Mirroring of images on `gcr.io`
+
+Due to Docker Hub's [rate-limiting policy](/blog/2020/docker-rate-limit/), the Istio community now also publishes
+images on `gcr.io/istio-release`. If you are experiencing issues with failed downloads in your environment, please set
+the `hub` in your `IstioOperator` or `istioctl` accordingly. For example, `istioctl install --set hub=gcr.io/istio-release`.
```

Comment:
```suggestion
To prevent our users from getting affected by Docker Hub's [rate-limiting policy](/blog/2020/docker-rate-limit/),
we are now publishing all our images on the `gcr.io/istio-release`registry. You can now set the hub in your installation step to `gcr.io/istio-release` to get around issues related to failed image downloads from Docker hub. Note that Docker hub is still the default hub for Istio installation.
```